features/step_definitions/api-token.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const Assert = require('chai').assert;
+const jwt = require('jsonwebtoken');
+const request = require('../support/request');
+const testTokenName = 'an access token';
+
+module.exports = function server() {
+    this.When(/^a new API token is generated$/, () =>
+        // Make sure there won't be a conflict: delete the token if it's already there
```

Comment:
Ooh, before hooks can be applied only to specific tags, that's probably cleaner. I'll do that.